packages/mermaid/src/docs/ecosystem/integrations-community.md

```diff
@@ -65,6 +65,8 @@ To add an integration to this list, see the [Integrations - create page](./integ
   - [Markdown for mermaid plugin](https://github.com/jamieh-mongolian/markdown-for-mermaid-plugin)
   - [redmine-mermaid](https://github.com/styz/redmine_mermaid)
 - Visual Studio Code [Polyglot Interactive Notebooks](https://github.com/dotnet/interactive#net-interactive)
+- LLM integrations
+  - [HueHive - Create mermaid diagrams with text](https://huehive.co/tools/diagrams)
```

Comment:
@kamalkishor1991 can you add LLM Integrations as a new headline please?

```suggestion
### LLM integrations

- [HueHive - Create mermaid diagrams with text](https://huehive.co/tools/diagrams)
```

Reply:
Done